src/app/services/client/client.service.ts

```diff
@@ -103,7 +103,12 @@ export class ClientService {
     });
   }
   getAgencies(): Observable<Agency[]> {
-    let url = this.BASE + `agencies?size=1000&api_key=${this.KEY}`;
+    let url = this.BASE + `agencies?size=1000`;
+
+    if(this.KEY) {
+      `&api_key=${this.KEY}`;
```

Comment:
@froi , maybe I'm reading this wrong, but shouldn't it be
```url += `&api_key=${this.KEY}`;```

Reply:
Yes. I forgot to put WIP in the title and I missed that.